docs/6.coordination/3.statistics.md

```diff
@@ -0,0 +1,124 @@
+## FIWARE Lab Statistics
+
+### Available Resources
+
+#### Physical resources {#physical-resources .ListParagraph}
+
+According to OpenStack documentation the default overcommit for RAM is
+1.5:1, the default overcommit for CPU is 16:1 and disk space for the
+Virtual Machines should not be overcommitted (please, notice that
+Volumes provisioned by Cinder is not physically where the Virtual
+machines are) -- That is shown in this page of the documentation:
+
+[*https://docs.openstack.org/arch-design/design-compute/design-compute-overcommit.html*](https://docs.openstack.org/arch-design/design-compute/design-compute-overcommit.html)
```

Comment:
Overcommitting CPU and RAM

`[*Overcommitting CPU and RAM*](https://docs.openstack.org/arch-design/design-compute/design-compute-overcommit.html)`